src/test/mochitest/browser_dbg-searching.js

```diff
@@ -2,12 +2,15 @@
  * http://creativecommons.org/publicdomain/zero/1.0/ */
 
 // Testing source search
-add_task(function* () {
+add_task(function*() {
   const dbg = yield initDebugger("doc-script-switching.html");
 
   pressKey(dbg, "sourceSearch");
   yield waitForElement(dbg, "input");
-  findElementWithSelector(dbg, "input").focus();
+  ok(
```

Comment:
yep, that will work!

i guess i was assuming it because the next action is typing... and it will work or fail based on that, but i don't care. this is nice to have.